__tests__/app.test.js

```diff
@@ -93,16 +105,54 @@ describe('/api/articles', () => {
     })
 })
 
-describe('/api', () => {
-    test('GET 200: returns a JSON object representing the the available endpoints of the API and their functionality', () => {
+describe('/api/articles/:article_id/comments', () => {
+    test('returns an array of all comments for the article with the given article_id', () => {
         return request(app)
-        .get('/api')
+        .get('/api/articles/1/comments')
         .expect(200)
         .then((result) => {
-            const {endpoints} = result.body
-            expect(endpoints).toEqual(api_endpoints)
+            const { comments } = result.body
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number)
```

Comment:
You're not expecting any article_id, you're specifically requesting comments for article 1